keylime/mba/elparsing/tpm2_tools_elparser.py

```diff
@@ -0,0 +1,264 @@
+import base64
```

Comment:
Why moving TPM related stuff from keylime.tpm to keylime.mba.elparsing?

Reply:
OK. So the goal is to remove the event log parser from the TPM class, that happens to use tpm2-tools for doing that for this implementation.

Maybe we should not reference tpm2_tools in the module name, then? Seems that the use of tpm2_eventlog is in this case a technical detail. Replacing the implementation to not use the tpm2_tools should not imply a renaming of the module and a change in the code that import it.

Reply:
If I understood your argument correctly @aplanas, I believe the question here is whether or not we are going to keep the invocation of the `tpm2-tools` eventlog parser as a possible "plugin" or not. This is a separate and interesting question, one that I do not believe we can collectively answer today.

Be as it may, even if we decide to keep it around, we should see something like both `tpm2_tools_elparser.py` and `native_python_elparser.py` (just invented the name for the sake of argument) on the aforementioned directory in a **future** PR (with the appropriate `importlib` code, of course!) and thus I favor that we merge "as is" now (there is third PR coming right after this one, FYI).

Reply:
My $0.02: we should only have one and not two since if we had two we would have to test two.

Reply:
I tend to agree with you. I believe our whole effort of avoiding installing `tpm2-tools` is a worthy one, but even if the majority of admins decide otherwise, we would be covered by having a "modular" implementation (in subsequent PRs, of course)

Reply:
I agree with @stefanberger also. The native event log parser is coming to keylime, and I think the incremental price in upkeep to for the parser is worth paying to be free of this: https://github.com/keylime/keylime/blob/be94e4f2ab2002155908f045ef99de86492236e9/keylime/tpm/tpm_main.py#L64-L84